### PR TITLE
React capstone project - MET ART APP

### PR DESCRIPTION
**Pull request by JuanDBta**
This React capstone project is about building a mobile web called MET ART APP to check a list of artists from the Metropolitan Museum of Art API making use of React and Redux.

The webapp will have several pages:

Home Page or Artists Page display a list of top nineteenth-century artists with their names and metrics of the number of paintings available in the MET.
Paintings Page is only displayed when you click the image of the artist you like and allows you to check the artists' paintings with specific details like the primary image, title, year of creation, and medium.
My Collection is an additional feature of the web app that allows you to select your favorite paintings on an independent page if you want to know more about each piece of art.

As requested the app follows these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), including:
Colors (select one main color for your website).
Typography: font face, size and weight.
Layout: composition and space between elements.
Original design idea by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio).

Finally you can find the following components:
Artists.jsx
Paintings.jsx
MyCollection.jsx
NavBar.jsx
selectedArtists.jsx and the Redux slices:

artistsSlice.js and paintingsSlice.js.

Added tests folder and styles modules.
Follow GitFlow, deploy project in Render and add Loom Video with the project's description.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -12,10 +14,17 @@ function App() {
...
+          <h1 className="title gill">MET</h1>
         </div>
-        <NavBar />
+        <NavBar className="bar" />
+        <CiBluetooth className="bluetooth" />
+        <p className="level">100%</p>
+        <CiBatteryFull className="battery" />
       </header>
       <Routes>
         <Route path="/" element={<Artists />} />
```
Comment (TadesseDev) [inline src/App.js]: - [ ] Dear @JuanDBta, for the homepage, we need to have a Home page that has a list of categories that could be `filtered using given parameters`, e.g. category name. In your case, I can see you have a list of artists, but there is no way to filter this item based on any specific attribute. Please make sure there is a way to filter elements inside the homepage.

File: src/App.js [hunk 2]
```diff
@@ -12,10 +14,17 @@ function App() {
     <>
       <header className="App-header">
         <div className="logo-title">
+          <RiFlickrFill className="icon1" />
+          <RiFlickrLine className="icon2" />
```
Comment (TadesseDev) [inline src/App.js]: ##### Dear @JuanDBta, for the layout of the project we are required to follow the design guideline closely and your design is missing some points despite it looking great we still need to make some changes. 
- [ ] For the navbar, some of the design belongs to the end device (mobile) we are going to use to open the page and should not be part of the webpage. this includes `the wifi indicator`, `the battery indicator `, `The network indicator`, and more. Please ignore this part of the design. 
- [ ] Notice the image on the original design are background images, the text goes over the image, and in your design, this is an image of their own and the text is on the other side. This is against the design guidelines. 
- [ ] On the original design, the items are aligned in `two columns` in a `single row` each containing their own item. In your carse every row contains a single column, Please follow the design guideline and arrange them in two columns. 


### Your design 
Screenshot from 2023-07-22 12-41-28

### The original design 
Screenshot from 2023-07-22 12-43-56

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @JuanDBta 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## Highlights
 - [x] Good job overall following the template design ✅
 - [x]  Most of the requirement has been meet. ✅
 - [x]  Good PR title/description  ✅

 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [Optional] I can see some of your changes are already on the base ranch. This is not a good idea as it will make your base branch inconsistent, beside that your code reviewer will not be able to notice all the changes. Please keep every little change in the feature branch for every project you are going to be working on. 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 2 above)
Comment (Whoistolu) [reply src/App.js]: - Well resolved 👍🏽

File: src/App.js (hunk 1 above)
Comment (Whoistolu) [reply src/App.js]: - Well resolved 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @JuanDBta 

Great job fixing the recommended changes by the previous reviewer 👍🏽 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**